pkg/operator/ceph/controller/spec.go

```diff
@@ -670,15 +670,17 @@ func LogCollectorContainer(daemonID, ns string, c cephv1.ClusterSpec) *v1.Contai
 		maxLogSize = resource.MustParse(fmt.Sprintf("%dM", size))
 	}
 
-	periodicity := "daily"
-	if c.LogCollector.Periodicity == "1h" {
+	var periodicity string
+	if c.LogCollector.Periodicity == "1h" || c.LogCollector.Periodicity == "hourly" {
 		periodicity = "hourly"
-	} else if strings.Contains(c.LogCollector.Periodicity, "h") || strings.Contains(c.LogCollector.Periodicity, "d") {
```

Comment:
with this check problem was, if user give `hourly` it will be valid here as will set periodicity to `daily` instead of `hourly`